src/id128/id128.c

```diff
@@ -125,6 +125,20 @@ static int verb_show(int argc, char **argv, void *userdata) {
                         r = show_one(&table, e->name, e->uuid, e == gpt_partition_type_table);
                         if (r < 0)
                                 return r;
+
+                        /* The DPS says that the UUID for /var/ should be keyed with machine-id, so
+                         * as a special case also show "var-uuid" after "var". */
+                        if (e->designator == PARTITION_VAR) {
```

Comment:
We should refuse to use/show GPT types if arg_app is specified. It would go awfully wrong especially if var-uuid shall be shown.

I.e. arg_app shall only be used with explicit uuid.

Reply:
I don't follow? I'm not using arg_app here?

Reply:
show_one might make use of that internally.

-> #33742